cardano-db-sync/app/cardano-db-sync.hs

```diff
@@ -230,20 +231,28 @@ pSlotNo =
           <> Opt.metavar "WORD"
       )
 
-pKeepTxMetadata :: Parser [Word64]
-pKeepTxMetadata =
+pWhiteListTxMetadata :: Parser [Word64]
+pWhiteListTxMetadata =
   Opt.option
     (parseCommaSeparated <$> Opt.str)
-    ( Opt.long "keep-tx-metadata"
+    ( Opt.long "whitelist-tx-metadata"
         <> Opt.value []
         <> Opt.help "Insert a specific set of tx metadata, based on the tx metadata key names"
     )
-  where
-    parseCommaSeparated :: String -> [Word64]
-    parseCommaSeparated str =
-      case traverse readMaybe (splitOn "," str) of
-        Just values -> values
-        Nothing -> error "Failed to parse comma-separated values"
+
+pWhiteListMAPolicies :: Parser [Word64]
+pWhiteListMAPolicies =
+  Opt.option
+    (parseCommaSeparated <$> Opt.str)
+    ( Opt.long "whitelist-multi-asset-policy"
```

Comment:
I would also put an `Opt.value` here to prevent this option from being mandatory

Reply:
I've deleted these because they are being replaced by your config file PR 😄